src/Uno.UI/UI/Xaml/Controls/Border/BorderLayerRenderer.Android.cs

```diff
@@ -474,6 +475,8 @@ public LayoutState(Windows.Foundation.Rect area, Brush background, Thickness bor
 
 				BackgroundColor = (Background as SolidColorBrush)?.Color;
 				BorderBrushColor = (BorderBrush as SolidColorBrush)?.Color;
+
+				BackgroundFallbackColor = (Background as XamlCompositionBrushBase)?.FallbackColor;
```

Comment:
A helper has been lot a while ago for that: `Brush.TryGetColorWithOpacity()`

@davidjohnoliver